src/cntk-train/src/main/python/CNTKLearner.py

```diff
@@ -8,11 +8,31 @@
 
 from mmlspark._CNTKLearner import _CNTKLearner
 from mmlspark.CNTKModel import CNTKModel as CNTKmod
-from pyspark.ml.common import inherit_doc
+from pyspark import SparkContext
+from pyspark.ml.common import inherit_doc, _java2py
 
 @inherit_doc
 class CNTKLearner(_CNTKLearner):
     """
+    Transforms the params from java object, with special case for byte array
+
+    Args:
+        java_model (CNTKModel): The python CNTKModel to transfer parameters to from java
+    """
+    def _custom_transfer_params_from_java(self, model):
```

Comment:
discussed, the code is a bit more convoluted than we thought, we will keep this as is for now